gridfs/gridfs.c

```diff
@@ -444,7 +444,7 @@ PHP_METHOD(MongoGridFS, storeBytes)
 
 	/* merge extra & zfile and add _id if needed */
 	zid = setup_extra(zfile, extra TSRMLS_CC);
-	setup_file_fields(zfile, NULL, bytes_len TSRMLS_CC);
+	setup_file_fields(zfile, NULL, (long) bytes_len TSRMLS_CC);
```

Comment:
`bytes_len` is an int populated by zpp, so I don't believe it's a candidate for change.